cmd/crates/soroban-spec-typescript/src/project_template/src/assembled-tx.ts

```diff
@@ -0,0 +1,706 @@
+import {
+  Account,
+  Address,
+  Contract,
+  Keypair,
+  Operation,
+  Server,
+  SorobanRpc,
+  StrKey,
+  TimeoutInfinite,
+  TransactionBuilder,
+  assembleTransaction,
+  hash,
+  nativeToScVal,
+  xdr,
+} from "soroban-client";
+import type { Memo, MemoType, Transaction } from "soroban-client";
```

Comment:
we should definitely have another PR either after this or as part of it to port to [`stellar-sdk`](https://github.com/stellar/js-stellar-sdk/releases/tag/v11.0.0-beta.6), to futureproof the bindings (the migration guide is [here](https://gist.github.com/Shaptic/5ce4f16d9cce7118f391fbde398c2f30)).

Reply:
For sure! I would like to do it after.